src/frontend/Komponenter/Behandling/Simulering/Tilbakekreving.tsx

```diff
@@ -74,16 +70,14 @@ export const Tilbakekreving: React.FC = () => {
         axiosRequest<ITilbakekreving, null>({
             method: 'GET',
             url: `familie-ef-sak/api/tilbakekreving/${behandlingId}`,
-        }).then((respons: Ressurs<ITilbakekreving>) => {
-            if (respons.status === RessursStatus.SUKSESS && respons.data) {
-                settBegrunnelse(respons.data.begrunnelse);
-                settTilbakekrevingsvalg(respons.data.valg);
-                settVarseltekst(respons.data.varseltekst || '');
-            } else if (
-                respons.status === RessursStatus.FEILET ||
-                respons.status === RessursStatus.FUNKSJONELL_FEIL ||
-                respons.status === RessursStatus.IKKE_TILGANG
-            ) {
+        }).then((respons: RessursSuksess<ITilbakekreving> | RessursFeilet) => {
+            if (respons.status === RessursStatus.SUKSESS) {
+                if (respons.data) {
```

Comment:
Trenger man å sjekke for `respons.data` her?

Reply:
Ja, backend returnerer `Ressurs<TilbakekrevingDto?>` så kan være suksess selvom data er null.